packages/openops/src/lib/ai/providers/azure-openai.ts

```diff
@@ -2,11 +2,28 @@ import { createAzure } from '@ai-sdk/azure';
 import { LanguageModel } from 'ai';
 import { AiProvider } from '../providers';
 
+function isApimUrl(url?: string) {
```

Comment:
basically, users can use Apim (proxy service) in azure, we might also want to support this in our other Azure actions.